dashboard/app/views/api/hoc_courses_teacher_guides.html.haml

```diff
@@ -1,18 +1,18 @@
 .row{style: "margin: 0;"}
-  = render partial: 'shared/course_tall_block', locals: { id: Script::STARWARS_NAME, teacher_guide: "/hourofcode/starwars" }
-  = render partial: 'shared/course_tall_block', locals: { id: Script::FROZEN_NAME, teacher_guide: "/hourofcode/frozen" }
+  = render partial: 'shared/course_tall_block', locals: { id: Script::STARWARS_NAME, teacher_guide: CDO.code_org_url("/hourofcode/starwars") }
+  = render partial: 'shared/course_tall_block', locals: { id: Script::FROZEN_NAME, teacher_guide: CDO.code_org_url("/hourofcode/frozen") }
   = render partial: 'shared/course_tall_block', locals: { id: 'sports' }
-  = render partial: 'shared/course_tall_block', locals: { id: Script::FLAPPY_NAME, teacher_guide: "/hourofcode/flappy" }
+  = render partial: 'shared/course_tall_block', locals: { id: Script::FLAPPY_NAME, teacher_guide: CDO.code_org_url("/hourofcode/flappy") }
 
 .row{style: "margin: 0;"}
-  = render partial: 'shared/course_tall_block', locals: { id: Script::HOC_NAME, teacher_guide: "/hourofcode/hourofcode" }
-  = render partial: 'shared/course_tall_block', locals: { id: Script::INFINITY_NAME, teacher_guide: "/hourofcode/infinity" }
-  = render partial: 'shared/course_tall_block', locals: { id: Script::PLAYLAB_NAME, teacher_guide: "/hourofcode/playlab" }
-  = render partial: 'shared/course_tall_block', locals: { id: Script::ARTIST_NAME, teacher_guide: "/hourofcode/artist" }
+  = render partial: 'shared/course_tall_block', locals: { id: Script::HOC_NAME, teacher_guide: CDO.code_org_url("/hourofcode/hourofcode") }
+  = render partial: 'shared/course_tall_block', locals: { id: Script::INFINITY_NAME, teacher_guide: CDO.code_org_url("/hourofcode/infinity") }
+  = render partial: 'shared/course_tall_block', locals: { id: Script::PLAYLAB_NAME, teacher_guide: CDO.code_org_url("/hourofcode/playlab") }
+  = render partial: 'shared/course_tall_block', locals: { id: Script::ARTIST_NAME, teacher_guide: CDO.code_org_url("/hourofcode/artist") }
 
 .row{style: "margin: 0;"}
-  = render partial: 'shared/course_tall_block', locals: { id: 'text_compression' }
-  = render partial: 'shared/course_tall_block', locals: { id: 'hoc_encryption' }
-  = render partial: 'shared/course_tall_block', locals: { id: 'conditionals', teacher_guide: "/hourofcode/unplugged-conditionals-with-cards" }
+  = render partial: 'shared/course_tall_block', locals: { id: 'text_compression', teacher_guide: "https://curriculum.code.org/hoc/2016/1/" }
```

Comment:
These two activities were missing the teacher links. They're on curriculum builder though, so I don't think we can keep `CDO.code_org_url` in the course_tall_block view?

Reply:
I think you're right. We don't have a url helper for curriculum builder (that I know of).

Reply:
ah, sorry about that! i only added teacher guides for courses that had links in the left-hand nav bar we removed

Reply:
No worries, Maddie. It'd be hard to know they existed!